oep-templates/external-link-template.rst

```diff
@@ -2,6 +2,8 @@
 
 .. Below is the display in the left sidebar on RTD. Please omit leading 0's
 
+.. _OEP XXXX OEP Template:
```

Comment:
I'm not sure why sphinx is not complaining about this, this label has been defined twice also it's only different from the label in `oep-templates/adr-based-template.rst` by one dash. Are these template files not processed by default or something?